string object_id
float32[3] pos
